README.md

```diff
@@ -1,33 +1,24 @@
-# Campmaster Constantine
+# DSC Bot
 
-[![builds.sr.ht status](https://builds.sr.ht/~muirrum/campmasterconstantine/.build.yml.svg)](https://builds.sr.ht/~muirrum/campmasterconstantine/.build.yml?)
-
-The Campmaster is designed to be a general-purpose Discord bot, that brings a message points/leveling system and a strike system to make moderation easier.
+The DSC Bot is designed to assist with moderation across server by having a shared banlist, shared warnings, and synced role information, along with a strike system and wordfilter.
 
 ## Features
-- Points gain based on messages
-    - Levels based on slow exponential (1.3x) cost to next level
-    - Level-based role rewarads*
 - Moderation
-    - Strikes as warnings
-    - Modification of a strike's reason
-    - Withdrawal of strikes
-    - Automatic banning/kicking of users based on strike count*
-    - Global and local message filter
-    - Moderation log for strikes and wordfilter matches
-- Question of the Day
-    - Suggestions
-    - Automatic ping of a role
+    - Strikes
```

Comment:
Could change this to "Server-specific Strikes and DSC-wide Advisories"

Reply:
to clarify - this isn't the reason I requested changes, and not resolving this won't block the PR, but I'm thinking we could clarify the distinction b/t strikes and advisories more